src/server/data/automatedTestingData.js

```diff
@@ -273,24 +400,24 @@ async function generateTestingData() {
 	// Generates 1 year of sinusoidal data with data points at 4-hour intervals
 	await generateFourHourTestingData();
 
-	// Generates 1 year of sinusoidal data with data points at 23-minute intervals
+	// // Generates 1 year of sinusoidal data with data points at 23-minute intervals
 	await generateTwentyThreeMinuteTestingData();
 
-	// Generates 1 year of sinusoidal data with data points at 15-minute intervals
+	// // Generates 1 year of sinusoidal data with data points at 15-minute intervals
 	await generateFifteenMinuteTestingData();
 
-	// Generates 1 year of sinusoidal data with data points at 1-minute intervals.
-	// Normally not desired so commented out.
-	// generateOneMinuteTestingData();
+	// // Generates 1 year of sinusoidal data with data points at 1-minute intervals.
+	// // Normally not desired so commented out.
+	// // generateOneMinuteTestingData();
 
-	// Generates 1 year of cosinusoidal data with an amplitude of 3 and with data points at 23-minute intervals.
-	// Should be related to 23-minute sinusoidal above.
+	// // Generates 1 year of cosinusoidal data with an amplitude of 3 and with data points at 23-minute intervals.
+	// // Should be related to 23-minute sinusoidal above.
 	await generateCosineTestingData(23, 3);
 
-	// Generates 2 years of *squared* sinusoidal data with an amplitude of 2.5 and with data points at 1-day intervals.
+	// // Generates 2 years of *squared* sinusoidal data with an amplitude of 2.5 and with data points at 1-day intervals.
 	await generateSineSquaredTestingData(2.5);
 
-	// Generates 2 years of *squared* cosinusoidal data with an amplitude of 2.5 and with data points at 1-day intervals.
+	// // Generates 2 years of *squared* cosinusoidal data with an amplitude of 2.5 and with data points at 1-day intervals.
 	await generateCosineSquaredTestingData(2.5);
 }
 
```

Comment:
Should say goes into meters and not file.

Not your doing but the following comment should say 1 to 3 not 1 to 7 and the description to match.